go/src/koding/kites/kloud/main.go

```diff
@@ -55,6 +55,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer k.Close()
```

Comment:
Just a note.

If statsd package does a cleanup in its Close method (like flush events etc.), it may not work when kloud is stopped / restarted by systemd - once go program receives a signal it executes a os.Exit(0), which do not run deferred methods.